src/libcrun/cgroup-systemd.c

```diff
@@ -302,7 +302,7 @@ systemd_job_removed (sd_bus_message *m, void *userdata, sd_bus_error *error arg_
     {
       d->terminated = 1;
       if (strcmp (result, "done") != 0)
-        crun_make_error (&d->err, 0, "error `%s` systemd unit `%s`: got `%s`", d->op, unit, result);
+        return crun_make_error (&d->err, 0, "error `%s` systemd unit `%s`: got `%s`", d->op, unit, result);
```

Comment:
could we make it clearer that this is a callback not handled by crun so not returning the value of `crun_make_error`.

Could we do?

```suggestion
  {
    crun_make_error (&d->err, 0, "error `%s` systemd unit `%s`: got `%s`", d->op, unit, result);
    return -1;
  }    
```

Reply:
That make sense